packages/financial-templates-lib/src/price-feed/EthVixPriceFeed.js

```diff
@@ -0,0 +1,114 @@
+const assert = require("assert");
+const moment = require("moment");
+const { parseFixed } = require("@uma/common");
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+
+// An implementation of PriceFeedInterface that uses the dVIX API to retrieve ethVIX prices.
+class ETHVIXPriceFeed extends PriceFeedInterface {
```

Comment:
Would it be useful to parameterize the identifier the VIX is calculated on? For example, this feed is ETH specific. it might be useful to refactor this to have a separate param that is `ETH` or whatever you want to calculate VIX on.

This might be useful seeing you are planning on adding other vix identifiers in the future.

Reply:
The main concern there is we're not entirely set on using this exact formula for future feeds. For instance, this feed relies on Deribit data and we feel an urgent need to move toward an entirely decentralized approach. For that reason we think keeping this explicitly the `ethVIX/iethVIX` feed rather than the `*VIX` price feed is more appropriate.

Reply:
To add to that, we would leave it up to the community to determine whether we continue to support the feed based on this model-free approach which has more centralized reliance than we'd probably like to see in DeFi. It is possible we forever continue using this exact approach, but it's equally possible a new and more decentralized source of truth comes available.

Reply:
I would suggest that the UMIP could be updated if the dVIX community chooses a different method for calculating VIX for ETH or for other assets.

This would be similar, in my opinion, to how UMIPs can be updated to reference different exchanges to get asset prices depending on changes in exchange volume or reliability. It's just more complicated, since the methodology changes may be substantially more than, "look at these other exchanges instead, and maybe take a median of three exchanges instead of a Uniswap TWAP," or whatever.

I don't think it's critical for this initial UMIP, but worth considering for a future UMIP that could add support for multiple asset VIX's through parameterization, especially as you evaluate how this methodology performs!

That future UMIP could include some description of how the methodology could change if the dVIX community decides there's a better way to calculate VIX for an asset, and that what you're relying on when using this price identifier in the general case is the dVIX community's judgement of how to calculate VIX, as modified by the DVM's judgement if someone thinks the dVIX community has gone rogue in some way.

Reply:
I think these are good points, perhaps the best thing to do is stick with this model for now and update the UMIP later if there are more true VIX-style indexes published. Can we defer this to a later update?

Reply:
For sure, I think that makes sense for now.